### PR TITLE
Invert matrix attempt2

### PR DESCRIPTION
**Pull request by SamCox822**

### CODE REVIEW

**Comment by whitead** [comment]
@SamCox822 almost there. Could you please get the case/name consistent between your folder name and other files? Thanks <3

File: data/Invert_Matrix_attempt2/Invert_Matrix_attempt2.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+
+import numpy as np
+
+x = np.array([[1,-1], [2,6]])
+m_i = invert_matrix(x)
+x_i = np.array([[0.75,0.125], [-0.25,0.125]])
+
+print (x)
+result = True if np.array_equal(m_i,x_i) == True else False
```
Comment (hgandhi2411) [inline data/Invert_Matrix_attempt2/Invert_Matrix_attempt2.py:10]: `np.array_equal` returns True or False. So, you can skip `True if np.array_equal() == True else False`

File: data/Invert_Matrix_attempt2/Invert_Matrix_attempt2.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+
+import numpy as np
+
+x = np.array([[1,-1], [2,6]])
+m_i = invert_matrix(x)
+x_i = np.array([[0.75,0.125], [-0.25,0.125]])
+
+print (x)
```
Comment (hgandhi2411) [inline data/Invert_Matrix_attempt2/Invert_Matrix_attempt2.py:9]: Can you take out the print statement?

**Review by hgandhi2411 — CHANGES_REQUESTED** [review]

**Review by hgandhi2411 — APPROVED** [review]
LGTM

**Pull request merged**